src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -175,26 +194,10 @@ module BABYLON {
         }
 
         public useStandingMatrix = (callback = (bool: boolean) => { }) => {

```

Comment:
is there a way to disable this? maybe make the flag public (or its getter and setter) and have this function simply change it? this way people won't really use this function and it can be deprecated.

Reply:
A function with a callback was used instead of  a flag so the developer could wait until the standing matrix is set before trying to use it.